packages/nouns-contracts/test/foundry/governance/fork/NounsDAOLogicV1Fork.t.sol

```diff
@@ -332,7 +361,7 @@ contract NounsDAOLogicV1Fork_DelayedGovernance_Test is ForkWithEscrow {
         token.setApprovalForAll(address(dao), true);
 
         dao.quit(tokens);
-        assertEq(proposer.balance, 10 ether);
+        assertEq(proposer.balance, 5 ether);
```

Comment:
@eladmallel maybe add to this flow that the other token holder can now claim and quit and get their 5 eth?